### DISCUSSION TITLE
Cannot find base config file "@vben/ts-config/vue-app.json" [tsconfig.json]

### DISCUSSION THREAD

**Discussion by 775100939** [vbenjs/vue-vben-admin#4359] [Q&A]
node版本：18.12.1
pnpm安装
pnpm dev运行时
![ffffffffffffffff](https://github.com/user-attachments/assets/7f15d769-41cb-44a0-b37f-cc4fbe3a1891)

**Comment by wangz-code** [comment]
最低貌似是需要  node  20.14.0

**Reply by 775100939** [reply]
我试试，
但是vben-admin-thin-next-main 这个项目也是一样的node版本，也跑起来了
